a-tier-bonding/src/components/header/header.js

```diff
@@ -49,7 +50,7 @@ function Header() {
         <img src={earthLogo} style={{ height: "75px", width: "75px" }} alt="Logo"/>
         <ButtonGroup color='inherit' size="large" variant="text">
           <Button component={Link} to='/'  style={{ fontSize: '2vh', fontFamily: "Nevis" }}>  Home  </Button>
-          <Button component={Link} to='/schedule' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Schedule  </Button> 
+          {/* <Button component={Link} to='/schedule' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Schedule  </Button>  */}
```

Comment:
Lets also remove the route too, so that people can't type "letsagame.com/schedule" to be brought to the unfinished schedule page.